test-runner/src/main/scala/com/typesafe/spark/test/mesos/mesosstate/StateDump.scala

```diff
@@ -1,15 +1,15 @@
-package mesostest.mesosstate
+package com.typesafe.spark.test.mesos.mesosstate
 
 import com.typesafe.config.Config
 import com.typesafe.config.ConfigFactory
 import java.net.URL
 import java.io.InputStreamReader
 
-import org.typesafe.spark.mesos.tests.MesosIntTestHelper
+import com.typesafe.spark.test.mesos.MesosIntTestHelper
 
 object MesosState extends Enumeration {
   type MesosState = Value
-  val TASK_RUNNING, TASK_FINISHED = Value
+  val TASK_RUNNING, TASK_FINISHED, TASK_STAGING = Value
```

Comment:
I think this is the [best we have](https://github.com/apache/mesos/blob/master/include/mesos/mesos.proto#L1074)